### ISSUE TITLE
Dota2 freezes with 100% CPU usage during Ranked gameplay

### ISSUE THREAD

**Issue by clintonshanewright** [ValveSoftware/steam-for-linux#5429]
#### Your system information

* Steam client version (build number or date): Mar 26 2018, 1522101301
* Distribution (e.g. Ubuntu): OpenSUSE Linux 42.2
* Opted into Steam client beta?: [Yes/No]
* Have you checked for system updates?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:
Dota 2 freezes up at 100% CPU usage (all I can see is a dump file if I run top that is using the 100% CPU. When I kill that process Dota 2 closes. I cannot find the dump file. Some help with that to better report my issue will be appreciated. I have since played games (because I am now penalized and placed in Low Priority) non ranked and against bots without it freezing.

I have not had this trouble before. No new system updates that have been introduced since the problem occured. Things have been running great with ZERO issues for months. Its only after I enabled DOTA PLUS that I encountered the issue of the game freezing.

When the game freezes, I have tried reconnecting to the game and it freezes after a minute or so again. I even tried restarting my computer, and reconnect again and it froze again after a minute.

#### Steps for reproducing this issue:

1. Install and run Dota2 on Steam Client for Linux
2. Enabled DOTA PLUS subscription
3. Play a standard ranked game for about 20 - 40 minutes

**Comment by kisak-valve** [comment]
Hello @clintonshanewright, let's track your issue in the Dota-2 issue tracker at https://github.com/ValveSoftware/Dota-2/issues/1429.
